From 9cec2fe3a4b7a9b7de468dc0de029b9ef4c8a1cf Mon Sep 17 00:00:00 2001
From: Iain Russell <Iain.Russell@ecmwf.int>
Date: Tue, 2 Aug 2022 14:39:12 +0100
Subject: [PATCH] ECC-1434: add shebang to grib_md5.sh

---
 tests/grib_md5.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/tests/grib_md5.sh b/tests/grib_md5.sh
index 41d4d5a29..810328076 100755
--- a/tests/grib_md5.sh
+++ b/tests/grib_md5.sh
@@ -1,10 +1,12 @@
+#!/bin/sh
+# (C) Copyright 2005- ECMWF.
 #
 # This software is licensed under the terms of the Apache Licence Version 2.0
 # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
 # 
 # In applying this licence, ECMWF does not waive the privileges and immunities granted to it by
 # virtue of its status as an intergovernmental organisation nor does it submit to any jurisdiction.
-#
+
 
 . ./include.ctest.sh
 
-- 
2.13.7

