diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS eccodes-2.28.0-Source.orig/src/grib_jasper_encoding.c eccodes-2.28.0-Source/src/grib_jasper_encoding.c
--- eccodes-2.28.0-Source.orig/src/grib_jasper_encoding.c	2023-02-22 13:24:02.387386179 +0100
+++ eccodes-2.28.0-Source/src/grib_jasper_encoding.c	2023-02-22 13:24:50.661619605 +0100
@@ -26,6 +26,7 @@
     int jaserr = 0;
     jas_conf_clear();
     jas_conf_set_max_mem_usage(jas_get_total_mem_size());
+    jas_conf_set_multithread(true);
     jaserr = jas_init_library();
     if (jaserr) return jaserr;
     jaserr = jas_init_thread();
